sdk/src/main/java/io/dapr/serializer/ObjectSerializerFactory.java

```diff
@@ -0,0 +1,10 @@
+package io.dapr.serializer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
```

Comment:
It seems that this is used only in the test, up to you, you can keep it or remove it and add it in the future, where we see a real need for this factory.